fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -259,13 +299,16 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 			}
 			a.result.send(result);
 		}
-	};
```

Comment:
end of indentation wonkiness.  I agree the `};` goes away.

Reply:
Fixed